hack/verify-container-images.sh

```diff
@@ -51,5 +57,21 @@ rm ${TOOL_BIN}/trivy.tar.gz
 ## Builds the container images to be scanned
 make REGISTRY=gcr.io/k8s-staging-cluster-api-aws PULL_POLICY=IfNotPresent TAG=dev docker-build
 
+BRed='\033[1;31m'
+BGreen='\033[1;32m'
+NC='\033[0m' # No
+
 # Scan the images
-${TOOL_BIN}/trivy image -q gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller-${GO_ARCH}:dev
\ No newline at end of file
+echo -e "\n${BGreen}List below dependencies that having fixes we can bump to fix the vulnerabilities:${NC}"
+${TOOL_BIN}/trivy image -q --exit-code 1 --ignore-unfixed --severity MEDIUM,HIGH,CRITICAL gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller-${GO_ARCH}:dev && R1=$? || R1=$?
+echo -e "\n${BGreen}List below dependencies including having fixes and no fixes for revew only:${NC}"
```

Comment:
```suggestion
echo -e "\n${BGreen}List of dependencies having fixes/no fixes for review only:${NC}"
```

Reply:
Thanks Ankita for suggestions, they are more accurate!

Updated the change: https://github.com/kubernetes-sigs/cluster-api-provider-aws/compare/c0fe14072c2bcc927b4619e34f45865ac1671df9..c2e1cbaf7dcc081a136c438dc18df8e1c8acf660